save-core/src/jvmMain/kotlin/org/cqfn/save/core/utils/ProcessBuilder.kt

```diff
@@ -12,17 +16,33 @@ actual class ProcessBuilder {
     private val pb = ProcessBuilder()
 
     actual fun exec(command: List<String>, redirectTo: Path?): ExecutionResult {
+        val common = ProcessBuilderInternal()
```

Comment:
Can it be a property of expect class?

Reply:
Expected property can't be initialized. What about creation `val common` without initialization - I don't know how reasonable is this. The code will be the same in `exec` function, instead of word `val`

Reply:
I think we should actually do vice-versa: have ordinary `class ProcessBuilder` in common sources along with `expect class ProcessBuilderInternal`; `ProcessBuilder` can have a property of type `ProcessBuilderInternal` and call it's methods inside common logic of `exec`